lib/puppet/cloudpack.rb

```diff
@@ -38,6 +38,9 @@ def to_hash
 
   require 'puppet/cloudpack/installer'
   class << self
+    if Puppet.respond_to?(:pe_version) && Puppet.pe_version.start_with?("3.8")
+      Puppet.warning "Cloud Provisioner is deprecated in PE 3.8. For more information and recommendations, see the release notes documentation here: https://docs.puppetlabs.com/pe/latest/release_notes.html"
```

Comment:
the URL should be `https://docs.puppetlabs.com/pe/3.8/release_notes.html` instead of /latest - that way if they are still on 3.8 when PE 4.0 is out, they are not seeing release notes for 4.0.